controllers/workspace/metrics/metrics.go

```diff
@@ -53,8 +65,7 @@ var (
 			Help:      "Number of failed DevWorkspaces",
 		},
 		[]string{
-			metricSourceLabel,
```

Comment:
It might still be useful to have `source` here (to e.g. distinguish between web terminal and Che) but I'm okay removing it for now.

Reply:
added back :+1: